engine/rpcserver.go

```diff
@@ -50,15 +51,18 @@ import (
 )
 
 var (
+	errExchangeNotEnabled   = errors.New("exchange is not enabled")
```

Comment:
This is unused, looks like youre using `errExchangeDisabled` instead, so this can be removed